allennlp/data/dataset_readers/snli.py

```diff
@@ -29,14 +29,27 @@ class SnliReader(DatasetReader):
         We use this `Tokenizer` for both the premise and the hypothesis.  See :class:`Tokenizer`.
     token_indexers : `Dict[str, TokenIndexer]`, optional (default=`{"tokens": SingleIdTokenIndexer()}`)
         We similarly use this for both the premise and the hypothesis.  See :class:`TokenIndexer`.
+    combine_input_fields : `bool`, optional
+            (default=`not isinstance(tokenizer, PretrainedTransformerTokenizer)`)
```

Comment:
This line is now wrong.  
```suggestion
            (default=`isinstance(tokenizer, PretrainedTransformerTokenizer)`)
```
After this, looks great!